app/boards/arm/puchi_ble/pinmux.c

```diff
@@ -11,9 +11,7 @@
 #include <zephyr/sys/sys_io.h>
 #include <zephyr/devicetree.h>
 
-static int pinmux_puchi_ble_init(const struct device *port) {
-    ARG_UNUSED(port);
-
+static int pinmux_puchi_ble_init() {
```

Comment:
I noticed that in other places you put `void` for empty parameters list (as is tradition), but not here. You might want to add it for consistency. Maybe that should be a pre-commit hook as well? Unsure, haven't used pre-commit before.

Reply:
Indeed. Good catch. Fixed.